stacks/core/dev/env.d/knot-cloud-storage.env

```diff
@@ -1,4 +1,4 @@
 DB_HOSTNAME=mongo
 DB_PORT=27017
 USERS_HOST=things
```

Comment:
@esdrasjnr  I would like to rename this to `THINGS_HOST` as I commented on your PR. So, maybe I could wait for it, right?